wallet/address.go

```diff
@@ -136,7 +142,7 @@ func (a AddressDec) Decode(r stdio.Reader) (err error) {
 func Key(a Address) AddrKey {
 	var buff strings.Builder
 	if err := io.Encode(&buff, a); err != nil {
-		panic("Could not encode address in AddrKey")
+		panic("Could not encode address in AddrKey" + err.Error())
```

Comment:
This will not print nicely. There should be a colon and a space.

Reply:
Updated in 7ebd50a9743a87e3a8bb063e02111377d4f87241.